build_tf.py

```diff
@@ -71,7 +71,7 @@ def main():
         os.chdir(pwd)
 
     use_intel_tf = False
-    if arguments.use_intel_tensorflow != '':
+    if arguments.use_intel_tensorflow:
         use_intel_tf = True
```

Comment:
This doesn't change the default. Please change the default to false.

Reply:
The action="strore_true" will set it as True only if the arg is specified